src/highlighter/css_highlighter.js

```diff
@@ -43,8 +43,25 @@ sre.CssHighlighter.prototype.highlightNode = function(node) {
     background: node.style.backgroundColor,
     foreground: node.style.color};
   var color = this.colorString();
-  node.style.backgroundColor = color.background;
   node.style.color = color.foreground;
+  if (node.bbox && node.bbox.w) {
+      var bbox = node.bbox;
+      var vpad = 0.05, hpad = 0; // vertical and horizontal padding
+      var span = document.createElement("span");
+      var left = parseFloat(node.style.paddingLeft||"0");
+      span.style.backgroundColor = c.background;
```

Comment:
Yes.  Sorry.  I was working from the minified version (which had that as `c`), and missed that one when converting to the actual file.  Thanks for catching it.